sites/default/menu_data.json

```diff
@@ -322,6 +322,14 @@
         "children" : [ ],
         "requirement" : 0,
         "acl_req" : [ "admin", "users" ]
+    }, {
+        "label" : "Users Activity",
+        "menu_id" : "adm0",
+        "target" : "adm",
+        "url" : "/interface/usergroup/users_activity.php",
```

Comment:
Please document this feature...somewhere.

Reply:
Need to test this again from a server and at least two client machines.  Honeypot Terry?

Reply:
Ok when .